src/components/Modal/Modal.jsx

```diff
@@ -1,14 +1,25 @@
 import { useState } from "react";
 
 import { useEscape } from "../../hooks";
+import { updateTask } from "../task/utils/task-utils";
+import { addTask } from "./utils/add-task";
 import { HiddenLabel } from "./HiddenLabel";
 
```

Comment:
Can u please remove remove these extra spaces as that code quality is good but the in between spaces can make it more readable. Logic is well implemented.

Reply:
Ok that explains.